plugin/audit_log/audit_log.c

```diff
@@ -1471,9 +1473,13 @@ static MYSQL_SYSVAR_STR(include_commands, audit_log_include_commands,
        audit_log_include_commands_update, NULL);
 
 static MYSQL_THDVAR_STR(local,
-                        PLUGIN_VAR_READONLY | PLUGIN_VAR_MEMALLOC | \
-                        PLUGIN_VAR_NOSYSVAR | PLUGIN_VAR_NOCMDOPT,
-                        "Local store.", NULL, NULL, "");
+       PLUGIN_VAR_READONLY | PLUGIN_VAR_MEMALLOC | \
+       PLUGIN_VAR_NOSYSVAR | PLUGIN_VAR_NOCMDOPT,
+       "Local store.", NULL, NULL, "");
+
+static MYSQL_THDVAR_ULONG(local_ptr,
```

Comment:
MySQL has its own static_assert, I'm not sure what's the exact name, maybe compile_time_assert